fix typos, wording, punctuation, other errors in docs

michizhou opened the pull request:
<!-- Thank you for submitting this PR! :) -->
<!-- Provide a general summary of your changes in the Title above, end with (close #<issue-no>) or (fix #<issue-no>) -->

### Description
<!-- Describe your changes in detail -->
I noticed there was potential for improvement in the docs of the GraphQL manual, with examples like:

- There is a typo in ``webhook.rst`` at https://github.com/hasura/graphql-engine/blob/master/docs/graphql/manual/auth/webhook.rst#L23
- There is an omission in ``index.rst`` at https://github.com/hasura/graphql-engine/blob/master/docs/graphql/manual/api-reference/index.rst#L19

There were other instances such as these in the docs which have been resolved by this PR.
<!-- Please put an `x` in the boxes below -->

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

### Related Issue
<!-- Please make sure you have an issue associated with this Pull Request -->
<!-- If you are suggesting a new feature or change, please discuss it in an issue first -->
<!-- If you are fixing a bug, there should be an issue describing it with steps to reproduce -->
<!-- Please don't forget to add `(close/fix #<issue-no>)` to the pull request title -->

<!-- Please link to the issue here: -->

### Solution and Design
<!-- How is this issue solved/fixed? What is the design? -->
<!-- It's better if we elaborate -->

### Type
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs update
- [ ] Community content

### Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [x] This change requires a change in the documentation. 
- [x] I have updated the documentation accordingly.
- [ ] I have added required tests.

shahidhk commented:
@rikinsk Needs review.

rikinsk commented:
@michizhou This looks good except for a couple of changes needed:
- There are changes to console files console/src/components/Main/Main.scss & console/src/components/Main/Main.css. Please revert them from this PR.
- In docs/graphql/manual/auth/roles-variables.rst, there is an extra comma after `Update` on line 119
- In docs/graphql/manual/mutations/multiple-mutations.rst, `in` would be the right word instead of `into` on line 6. The inserting is not into the mutation but rather using the same mutation for inserts. Hope that makes sense.

michizhou commented:
@shahidhk @rikinsk I have reverted the Main.scss and Main.css console files and changed the GraphQL manual documentation according to the comments of @rikinsk. Please feel free to let me know if there are any other changes that need to be made. Thanks!
